clients/common/rocblas_gentest.py

```diff
@@ -0,0 +1,255 @@
+#!/usr/bin/python
```

Comment:
This is the main engine for the creation of test cases from YAML files. It parses YAML files, handles !include directives from one YAML file to another, and generates binary files corresponding to Arguments structure records. It is fairly complex Python code, but it is hopefully readable.